Tools/Scripts/webkitpy/test/main.py

```diff
@@ -74,7 +74,6 @@ def main():
     if sys.platform != 'cygwin':
         tester.add_tree(os.path.join(_webkit_root, 'Source', 'WebKit', 'Scripts'), 'webkit')
 
-    tester.skip(('webkitpy.common.checkout.scm.scm_unittest',), 'are really, really, slow', 31818)
```

Comment:
So, the reason we skipped these tests is that they are very slow...they also aren't terribly useful, I would be against unsnapping them in general, they should be skipped for most local runs.

Reply:
It tooks only a minute. I don't think it's really slow.
I think using '--all' isn't a good idea because it runs platform specific tests.

Reply:
We no longer use svn. Can we remove `SVNTest` and `GitSVNTest` in scm_unittest.py? They are marked `@slow`.

Reply:
unittest has `@unittest.skipUnless` and `@unittest.skipIf`. https://docs.python.org/3.5/library/unittest.html
We can use `@unittest.skipUnless` and `@unittest.skipIf` to skip platform specific tests, and use `tester.skip` for slow tests.
Is this better idea?

Reply:
> I think using '--all' isn't a good idea because it runs platform specific tests.

Oh, does it unskip the platform-specific exclusions below? ergh.

Reply:
We can remove SVN tests, and probably `git` ones too, actually, since most of our `git` tooling lives in `webkitscmpy` now, which takes a different approach to testing than `webkitpy`

Reply:
That being said, feels like the most immediate task is just the revert, we can clean up in future changes.

Reply:
> Oh, does it unskip the platform-specific exclusions below? ergh.

It's not a problem for Mac port becuase Mac always runs Linux and Windows tests.
https://bugs.webkit.org/show_bug.cgi?id=222066

Reply:
I recreated the PR to revert 253880@main